GOT-StudentVersion/GOT-StudentVersion/GOTEpisode.swift

```diff
@@ -29,6 +29,18 @@ class GOTEpisode {
         self.mediumImageID = mediumImageID
         self.originalImageID = originalImageID
     }
+    
+    
+    static func filterAndStoreBySeason() -> [[GOTEpisode]] {
```

Comment:
ooohh I really like this function. Instead of filtering each one, you made it dynamic with a for loop. Good job